.gitignore

```diff
@@ -5,3 +5,4 @@ build
 dist
 liquidctl/extraversion.py
 .coverage
+env/**
```

Comment:
This PR is probably not the right place for this.

Additionally, since the venv directory name can be different (for example, I'm more used to seeing `venv`), depending on the developer, it's something more suitable for each developer's individual global gitignore file ([`core.excludesFile`](https://git-scm.com/docs/gitignore)).

If you don't mind, can you drop the commit that added this?

Reply:
yup, my bad. I just dumped it there not really thinking about it. I can fix that in the morning